app/src/main/java/com/example/movopfy/features/home/presentation/HomeScreen.kt

```diff
@@ -0,0 +1,105 @@
+package com.example.movopfy.features.home.presentation
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.LazyRow
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.CircularProgressIndicator
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.collectAsState
+import androidx.compose.runtime.getValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.text.font.Font
+import androidx.compose.ui.text.font.FontFamily
+import androidx.compose.ui.unit.dp
+import coil.compose.AsyncImage
+import com.example.movopfy.R
+import com.example.movopfy.features.home.domain.models.WaitingListToday
+import com.example.movopfy.main.ui.theme.BackgroundMain
+import com.example.movopfy.main.ui.theme.Dimensions
+import com.example.movopfy.main.ui.theme.LocalDim
+import com.example.movopfy.main.ui.theme.TextMain
+import org.koin.androidx.compose.koinViewModel
+
+@Composable
+fun HomeScreen(viewModel: HomeViewModel = koinViewModel()) {
+
+    val uiState by viewModel.uiState.collectAsState()
+
+    val dimensions = LocalDim.current
+
+    Column(
+        modifier = Modifier
+            .background(color = BackgroundMain)
+            .fillMaxSize()
+    ) {
+        TextWaitingListToday(dimensions = dimensions)
+
+        when (val state = uiState) {
+            is HomeViewModel.HomeUiState.Loading -> {
+                ProgressBarForWaitingListToday()
+            }
+
+            is HomeViewModel.HomeUiState.Loaded -> {
+                LazyRowWaitingListToday(list = state.schedules, dimensions = dimensions)
+            }
+        }
+    }
+}
+
+@Composable
+fun TextWaitingListToday(dimensions: Dimensions) {
+
+    Text(
+        text = "Расписание на сегодня",
```

Comment:
please extract to string resources

Reply:
I did name text_waiting_today_home_screen

Reply:
please rename to schedule_for_today_text